client/src/components/create-assignment/LeafAssignment.js

```diff
@@ -15,7 +15,7 @@ import StringTextField from './StringTextField';
 import DateTextField from './DateTextField';
 import assignmentServices from '../../services/assignments';
 
-// An Assignmnet component without subassignments and hence without a formula as well.
+// An Assignmnet component without subAttainments and hence without a formula as well.
```

Comment:
Still assignment written here

Reply:
I kept Assignment here because the component name hasn't been changed.